js/home.js

```diff
@@ -9,204 +9,247 @@ weekInfoContainer.className = "container";
 weekInfo.append(weekInfoContainer);
 
 inputField.addEventListener("keypress", (e) => {
-  if (e.key === "Enter") {
-    e.preventDefault();
-    if (e.target.value.trim() !== "") {
-      let searchTerm = e.target.value;
-      fetchWeatherData(searchTerm);
+    if (e.key === "Enter") {
+        e.preventDefault();
+        if (e.target.value.trim() !== "") {
+            let searchTerm = e.target.value;
+            fetchWeatherData(searchTerm);
+        }
     }
-  }
 });
 
 function fetchWeatherData(searchTerm) {
-  let lat,
-    lon,
-    countryName,
-    cityName,
-    currentDayData,
-    dailyData,
-    apiKey = "d0139168498c4f66d3fb31b4d374f145";
-  fetch(
-    `https://api.openweathermap.org/data/2.5/weather?q=${searchTerm}&appid=${apiKey}`
-  )
-    .then((res) => res.json())
-    .then((weatherInfo) => {
-      lat = weatherInfo.coord.lat;
-      lon = weatherInfo.coord.lon;
-      countryName = weatherInfo.sys.country;
-      cityName = weatherInfo.name;
-      fetch(
-        `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=minutely,hourly,alerts&appid=${apiKey}`
-      )
-        .then((info) => info.json())
-        .then((db) => {
-          currentDayData = db.current;
-          dailyData = db.daily.slice(1, 7);
-          renderCurrentDayData(currentDayData, countryName, cityName);
-          renderDailyData(dailyData);
-        });
-    })
-    // If the search term is INVALID, a popup will show to ask the user to re-write a VALID input
-    .catch((err) => placeNotFound(searchTerm));
+    let lat,
+        lon,
+        countryName,
+        cityName,
+        currentDayData,
+        dailyData,
+        apiKey = "d0139168498c4f66d3fb31b4d374f145";
+    fetch(
+            `https://api.openweathermap.org/data/2.5/weather?q=${searchTerm}&appid=${apiKey}`
+        )
+        .then((res) => res.json())
+        .then((weatherInfo) => {
+            lat = weatherInfo.coord.lat;
+            lon = weatherInfo.coord.lon;
+            countryName = weatherInfo.sys.country;
+            cityName = weatherInfo.name;
+            fetch(
+                    `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=minutely,hourly,alerts&appid=${apiKey}`
+                )
+                .then((info) => info.json())
+                .then((db) => {
+                    console.log(db)
+                        ////////////////////////////
+                    let currente = db.current.weather[0].main //give the weather condition 
+                        //images show According to the weather 
+                    let images = {
+                            Rain: ["../assets/images/rainy/1.jpg", "../assets/images/rainy/2.jpg", "../assets/images/rainy/3.jpg", "../assets/images/rainy/4.jpg"],
+                            Clear: "../assets/images/clear/1.jpg",
```

Comment:
Put `"../assets/images/clear/1.jpg"` inside an array, so the `randomImages()` function can choose a random image from the array